packages/lesswrong/components/localGroups/SmallMapPreview.tsx

```diff
@@ -67,7 +70,7 @@ class SmallMapPreview extends Component<SmallMapPreviewProps,SmallMapPreviewStat
         {...viewport}
         width="100%"
         height="100%"
-        mapStyle={"mapbox://styles/habryka/cilory317001r9mkmkcnvp2ra"}
+        mapStyle={isEAForum ? undefined : "mapbox://styles/habryka/cilory317001r9mkmkcnvp2ra"}
```

Comment:
For now I'm using Mapbox's default style for our maps, but I'd be happy to make a custom style if people have preferences.